wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/agent/server/WsAgentPingRequestFactoryTest.java

```diff
@@ -49,7 +49,7 @@
   //
   //        when(httpJsonRequestFactory.fromUrl(anyString())).thenReturn(httpJsonRequest);
   //        when(httpJsonRequest.setMethod(HttpMethod.GET)).thenReturn(httpJsonRequest);
-  //        when(server.getProperties()).thenReturn(serverProperties);
```

Comment:
@vparfonov can you take a look whether we still need this class?

Reply:
we can remove it

Reply:
I've remove it